packages/components/popper/src/content.vue

```diff
@@ -84,17 +84,23 @@ export default defineComponent({
         (val) => {
           updateHandle?.()
           if (val) {
-            popperInstanceRef.value?.destroy()
             const popperContentEl = unref(popperContentRef)!
             contentRef.value = popperContentEl
             const arrowEl = unref(arrowRef)
 
-            const newInstance = createPopperInstance({
-              referenceEl: val,
-              popperContentEl: unref(popperContentRef)!,
-              arrowEl,
-            })
-            popperInstanceRef.value = newInstance
+            watch(
+              () => props.placement,
+              () => {
+                popperInstanceRef.value?.destroy()
+                const newInstance = createPopperInstance({
+                  referenceEl: val,
+                  popperContentEl: unref(popperContentRef)!,
+                  arrowEl,
+                })
+                popperInstanceRef.value = newInstance
+              },
+              { immediate: true }
+            )
```

Comment:
For fixing this issue, I think we should update the instance, instead of creating a new one which could introduces additional overhead.

Reply:
You're right, this solution is better. 

We update the instance when the buildPopperOptions changed.

```js
 watch(() =>
    buildPopperOptions(props, {
        arrowEl: unref(arrowRef),
        arrowOffset: unref(arrowOffset),
    }),
    (option) => popperInstanceRef.value?.setOptions(option)
)
```